core/src/Properties/InternalsVisibleTo.cs

```diff
@@ -27,5 +27,8 @@
 
 using System.Runtime.CompilerServices;
 
-// The core test project is the only one allowed to reference Core.dll for convenience. Do not add any other entries here. 
+// The core test project is the only one allowed to reference Core.dll for convenience. Do not add any other entries here, except for test.
 [assembly: InternalsVisibleTo("Test.Microsoft.Identity.Core.Unit, PublicKey=0024000004800000940000000602000000240000525341310004000001000100B5FC90E7027F67871E773A8FDE8938C81DD402BA65B9201D60593E96C492651E889CC13F1415EBB53FAC1131AE0BD333C5EE6021672D9718EA31A8AEBD0DA0072F25D87DBA6FC90FFD598ED4DA35E44C398C454307E8E33B8426143DAEC9F596836F97C8F74750E5975C64E2189F45DEF46B2A2B1247ADC3652BF5C308055DA9")]
+//[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100B5FC90E7027F67871E773A8FDE8938C81DD402BA65B9201D60593E96C492651E889CC13F1415EBB53FAC1131AE0BD333C5EE6021672D9718EA31A8AEBD0DA0072F25D87DBA6FC90FFD598ED4DA35E44C398C454307E8E33B8426143DAEC9F596836F97C8F74750E5975C64E2189F45DEF46B2A2B1247ADC3652BF5C308055DA9")]
```

Comment:
>//[ [](start = 0, length = 3)

?

Reply:
Removed.

Reply:
I still see it :)

---
In reply to: [219531456](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1259#discussion_r219531456) [](ancestors = 219531456)